### PR TITLE
updated current files w/ year & created new jsons for 2016-2017 data

### PR DESCRIPTION
**Pull request by FishheadNate**
- Renamed original json files to include the year
- Create 2 new json files:
   "all-2016-2017.json" (to replace "all-2016.json")
   "wildlifeareas2017.json" (new data) 
- Added 2 fields to each json file:
   "year" (year the data was published)
   "source" (url for were the data was published)

### CODE REVIEW

**Comment by FishheadNate** [comment]
Do I need to change "all-2016-2017.json" to "all2016-2017.json"?

**Comment by JesseCrocker** [comment]
The error is because it wants the year field to have multiple years separated by `,`, but i think there are cases where `-` is correct. Is this one of those cases? If so i'll change the schema so thats valid.

**Comment by FishheadNate** [comment]
I think the `-` is valid because the data is labeled as "in effect for April 1 2016 to March 31 2017".

I couldn't find any data for April 1 2017 to March 31 2018 on the any of the WA sites

**Comment by JesseCrocker** [comment]
Sounds good, so leave it as is and i'll change the test so it passes

File: sources/US/WA/wildlife-areas-2017.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "name" : "WA Wildlife Areas",
+    "url": "http://geo.wa.gov/datasets/89e677d17858438b9befb2963f0bd92f_0.geojson",
+    "species": [
+       "all"
+    ],
+    "attribution": "Washington Department of Fish and Wildlife",
+    "properties": {
+       "id": "GMU_Num",
```
Comment (JesseCrocker) [inline sources/US/WA/wildlife-areas-2017.json]: It doesnt look like the properties in the file match the properties listed here. 
screen shot 2017-08-28 at 9 29 58 am

**Comment by dvdgldbch** [comment]
Appears the WA GMU layer is no longer functioning. Ideas?

**Pull request merged**